app/javascript/components/restaurants/FilterBar.js

```diff
@@ -0,0 +1,151 @@
+import React from "react"
+import DynamicBar from 'components/restaurants/DynamicBar'
+import {Icon} from 'react-icons-kit'
+import {filter, cross} from 'react-icons-kit/icomoon'
+
+class FilterForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            cuisineOptions: [],
+            tenBisOptions: ['', true, false],
+            ratingOptions: ['', 1, 2, 3, 4, 5]
+        }
+    }
+
+    render() {
+        return (
+            <div className="container">
+                <FilterOptions
+                    tenBisOptions={this.state.tenBisOptions}
+                    cuisineOptions={this.state.cuisineOptions}
+                    ratingOptions={this.state.ratingOptions}
+                    changeOption={this.props.onFilterChange}
+                    filteredValues={this.props.filteredValues}
+                    filteredList={this.props.filteredList}
+                />
+            </div>
+        )
+    }
+};
+
+let FilterOption = (id, value, options) => {
```

Comment:
shouldn't this be a const? 
also, it's great that you create a functional component but please separate it to its own file.